_browser/blockstack_storage.md

```diff
@@ -220,48 +220,36 @@ These are the `UserSession.putFile`, `UserSession.getFile`, and `lookupProfile`
 
 1. Open the `src/components/Profile.js` file.
 
-2. Replace the initial state in the `constructor()` method so that it holds the key properties required by the app.
+2. Replace the initial state in the component method so that it holds the key properties required by the app.
```

Comment:
I got lost here. What should be replaced? the part in the constructor or the whole component?

Reply:
This is very React-specific knowledge. TBH, I'm not sure how to put it. It's not a constructor/class anymore, it's a functional component, which is considered React best practice these days. You setup state using hooks in the beginning of the function.

Reply:
I think my main confusion came from the issue that I used the old generator. It didn't "fit" with what I was looking at